sunpy/database/database.py

```diff
@@ -154,10 +156,14 @@ class Database(object):
         The default value is :class:`sunpy.database.caching.LRUCache`.
     cache_size : int
         The maximum number of database entries, default is no limit.
-    default_waveunit : str, optional
+    default_waveunit : str or astropy.units.Unit, optional
         The wavelength unit that will be used if an entry is added to the
         database but its wavelength unit cannot be found (either in the file or
         the VSO query result block, depending on the way the entry was added).
+        If an astropy.units.Unit is passed, it is assigned to default_waveunit.
```

Comment:
same everywhere else, also default_waveunit should be in double backticks:

```
``default_wavelength``
```
so that it is formated varbatim.